vnext/Shared/Modules/WebSocketModule.cpp

```diff
@@ -50,7 +56,7 @@ std::vector<facebook::xplat::module::CxxModule::Method> WebSocketModule::getMeth
 {
   return
   {
-    Method(
+    {
       "connect",
       [this](dynamic args) // const string& url, dynamic protocols, dynamic options, int64_t id
```

Comment:
>this [](http://example.com/codeflow?start=7&length=4)

Don't you need to pass weak m_holder here?

Reply:
Possibly. This is work in progress. `WebSocketModule` will require some refactoring.